python/py-pythonz/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        berdario pythonz 1.11.4
+github.tarball_from releases
+name                py-${github.project}
+
+master_sites        ${github.homepage}/archive/
+distname            ${github.project}-${github.version}
+worksrcdir          ${github.project}-${github.project}-${github.version}
```

Comment:
The `distname` and `worksrcdir` lines can be deleted because they should be set correctly by the GitHub portgroup.

Reply:
I think I needed to edit the `distname` and `worksrcdir` since the release name on github is different from the standard one. Am I missing something?

Reply:
I'm pretty sure those are the normal and default values. Please try commenting out those lines and see if it works.

Reply:
I suspect you shouldn't use the line ```github.tarball_from releases``` as it doesn't look like upstream uses ```releases```. I think if you remove that line, the others can go as well as @mf2k suggested.

Reply:
I removed `github.tarball_from releases`, and `distname` ands `woirksrcdir` but now macports cannot find the right extracted directory...

Reply:
```
:info:build Executing:  cd "/opt/local/var/macports/build/_Users_*****_git_macports-ports_python_py-pythonz/py37-pythonz/work/pythonz-1.11.4" && /opt/local/Library/Frameworks/Python.framework/Versions/3.7/bin/python3.7 setup.py --no-user-cfg build
:debug:build system:  cd "/opt/local/var/macports/build/_Users_*****_git_macports-ports_python_py-pythonz/py37-pythonz/work/pythonz-1.11.4" && /opt/local/Library/Frameworks/Python.framework/Versions/3.7/bin/python3.7 setup.py --no-user-cfg build
:info:build /opt/local/Library/Frameworks/Python.framework/Versions/3.7/Resources/Python.app/Contents/MacOS/Python: can't open file 'setup.py': [Errno 2] No such file or directory
```

Reply:
```
ll /opt/local/var/macports/build/_Users_*****_git_macports-ports_python_py-pythonz/py37-pythonz/work/
total 0
drwxr-xr-x   2 macports  wheel    64B Apr 27 20:24 pythonz-1.11.4
drwxr-xr-x  12 macports  wheel   384B Feb  1  2017 pythonz-pythonz-1.11.4
```

`pythonz-1.11.4` is empty. As you can see the zip gets extracted in the directory `pythonz-pythonz-1.11.4`

Reply:
(And also the tarball)

Reply:
I see, while the `distname` should be set correctly by the GitHub portgroup, the `worksrcdir` does look non-standard.

Reply:
you should use ```github.setup        berdario pythonz 1.11.4 pythonz-``` instead; then you can get rid of the ```master_sites```, ```distnames``` and ```worksrcdir```. Also, please remove ```use_zip``` since there is also a ```.tar.gz``` file that is smaller in size.

Reply:
```